--- gtkspell3-3.0.1/gtkspell/Makefile.am.orig	2013-02-07 00:30:42.000000000 +0100
+++ gtkspell3-3.0.1/gtkspell/Makefile.am	2013-04-01 16:43:02.304629366 +0200
@@ -40,7 +40,7 @@
 if ENABLE_GTK3
 GtkSpell-3.0.gir: libgtkspell3-3.la
 GtkSpell_3_0_gir_INCLUDES = GObject-2.0 GLib-2.0 Gtk-3.0
-GtkSpell_3_0_gir_CFLAGS = --warn-all @GTK_SPELL_CFLAGS@
+GtkSpell_3_0_gir_CFLAGS = --warn-all @GTK_SPELL3_CFLAGS@
 GtkSpell_3_0_gir_LIBS = libgtkspell3-3.la
 GtkSpell_3_0_gir_FILES = $(libgtkspell3_3_la_SOURCES)
 INTROSPECTION_GIRS += GtkSpell-3.0.gir
